### PR TITLE
Add announcement for the QMSTR 0.5 requirements workshop.

### PR DESCRIPTION
**Pull request by mirkoboehm**

### CODE REVIEW

File: content/blog/qmstr-requirements-workshop-0.5.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+events, a hands-on hack day on April 3, and the actual requirements
+planning that will take place on April 4. A dinner will be organized
+for the evening of April 3 for all those who are interested.
+
+## April 3: Hack Day
+
+The intention of the hack day is to give participants a chance to work
+with the QMSTR development team, get demonstrations of recent
+features, try out writing an analyzer module or walk through details
+of setup and deployments, and egnerally to have a good time and get
```
Comment (lisa-noeth) [inline content/blog/qmstr-requirements-workshop-0.5.md]: generally

File: content/blog/qmstr-requirements-workshop-0.5.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+# QMSTR Requirements Workshops
+
+The goal of the workshop is to collaboratively develop the roadmap for
+QMSTR 0.5 together with interested industry and community
+contributors. It will serve to collect feedback as well as industry
+and community input on functionality needs. We intend the workshop to
+be a hands-on, collaborative experience, complementing the ongoing
+decentralized development on Github. There will be two parts to the
+events, a hands-on hack day on April 3, and the actual requirements
```
Comment (lisa-noeth) [inline content/blog/qmstr-requirements-workshop-0.5.md]: event

File: content/blog/qmstr-requirements-workshop-0.5.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+dinner-and-drinks get-together is planned for the evening.
+
+## April 4: Requirements Workshop
+
+April 4 is the day to review current and recently added functionality,
+and plan goals for the six development sprints leading to QMSTR
+version 0.5. There will be a lot to talk about with the [FASTEN
+Project](https://www.fasten-project.eu/) starting and the Linux
+Foundation ACT project launching soon. Besides technical discussions
+and roadmap planning, the idea is also to identify opportunities to
```
Comment (lisa-noeth) [inline content/blog/qmstr-requirements-workshop-0.5.md]: to also

File: content/blog/qmstr-requirements-workshop-0.5.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+To register, please use the contact form and state that you would like
+to attend the workshop. You will receive an email with a confirmation
+shortly, provided there are stil seats available. Registration is on a
+first-come-first-served basis, with a (somewhat flexible) upper limit
+of 20 participants. The workshop will take place at [Endocode's Berlin
+office](https://endocode.com/about/). The [Park Inn
+Alexanderplatz](https://www.parkinn.com/hotel-berlin) or the [Motel
+One
+Berlin-Mitte](https://www.motel-one.com/en/hotels/berlin/hotel-berlin-mitte/)
+are close by, as are many others many other hotels in the
```
Comment (lisa-noeth) [inline content/blog/qmstr-requirements-workshop-0.5.md]: many others many other

**Review by lisa-noeth — COMMENTED** [review]
Looks good to me.

**Review by GiasemiSh — APPROVED** [review]
Apart from what Lisa found lgtm

**Pull request merged**